WcaOnRails/config/locales/en.yml

```diff
@@ -623,7 +623,7 @@ en:
     can_register: "You can register for %{comp} below."
     check_registration_information: "Please check the information about registration to find out if you need to do something else, like paying a registration fee."
     have_registered: "You have registered for %{comp}. You can see your registration information below."
-    entry_fee: "The entry fee is "
+    entry_fee: "The entry fee is"
```

Comment:
Hmm... this assumes the the entry fee always comes at the end of the sentence. That might not be true in all languages, right?

I think the safest thing to do is to do string interpolation instead? Like this:

```
entry_fee: "The entry fee is %{fee}"
```

Then, the call to `t` in the view file will need to pass in the fee.